cpp/src/interop/to_arrow.cu

```diff
@@ -296,8 +297,13 @@ std::shared_ptr<arrow::Array> dispatch_to_arrow::operator()<cudf::string_view>(
     auto tmp_data_buffer                 = allocate_arrow_buffer(0, ar_mr);
     tmp_offset_buffer->mutable_data()[0] = 0;
 
-    return std::make_shared<arrow::StringArray>(
-      0, std::move(tmp_offset_buffer), std::move(tmp_data_buffer));
+    if (cudf::strings::detail::is_large_strings_enabled()) {
+      return std::make_shared<arrow::LargeStringArray>(
+        0, std::move(tmp_offset_buffer), std::move(tmp_data_buffer));
+    } else {
+      return std::make_shared<arrow::StringArray>(
+        0, std::move(tmp_offset_buffer), std::move(tmp_data_buffer));
+    }
```

Comment:
This should not be needed. I see no reason to create a separate empty LargeStringArray.